packages/framework/synthesize/api-report/synthesize.legacy.alpha.api.md

```diff
@@ -14,15 +14,15 @@ export type AsyncOptionalFluidObjectProvider<T> = T extends undefined ? Record<s
 
 // @alpha @legacy
 export type AsyncRequiredFluidObjectProvider<T> = T extends undefined ? Record<string, never> : {
-    [P in keyof T]: Promise<NonNullable<Exclude<T[P], undefined | null>>>;
+    [P in keyof T]: Promise<NonNullable<Exclude<T[P], undefined>>>;
```

Comment:
This is a breaking change, so we will need to undo this (and disable the linter rule at the relevant site)

Reply:
Gotcha. Thanks for clarifying!